target_chains/ethereum/contracts/contracts/pulse/Pulse.sol

```diff
@@ -29,6 +29,7 @@ abstract contract Pulse is IPulse, PulseState {
         _state.pyth = pythAddress;
         _state.currentSequenceNumber = 1;
         _state.defaultProvider = defaultProvider;
+        _state.exclusivityPeriodSeconds = 15; // Default to 15 seconds
```

Comment:
I recommend making it configurable in deployment as well. I think 15s is good but in some chains we might need faster and in some we need slower (like eth)